samples/keys/hw_unique_key/src/derive_key.c

```diff
@@ -29,7 +29,7 @@ LOG_MODULE_DECLARE(app, LOG_LEVEL_DBG);
 psa_status_t derive_key(psa_key_attributes_t *attributes, uint8_t *key_label,
 			uint32_t label_size, psa_key_id_t *key_id_out)
 {
-	uint8_t key_out[HUK_SIZE_BYTES];
+	uint8_t key_out[PSA_BITS_TO_BYTES(256)];
```

Comment:
Why do we hardcode the number here now instead of using the definition?

Reply:
it's the wrong definition to use. the length of the derived key is unrelated to the "huk-size" (which is 48 bytes on cracen. It just happened to align with aes key sizes on cryptocell)